content/en/about-filecoin/faq.md

```diff
@@ -136,7 +142,9 @@ There's a pretty large design space here, and we can do a bunch of different thi
 
 ### Who will be selected to be verifiers to verify clients on the network?
 
-We don't have more updates about verifiers at this time. We will post on our blog once we do.
+Notaries, selected through an application process, serve as fiduciaries for the Filecoin Network and are responsible for allocating DataCap to clients with valuable storage use cases.
+
+See [Filecoin Plus > Overview > Notary](https://docs.filecoin.io/store/filecoin-plus/#notary).
```

Comment:
Simply use the title of the page when linking to it. We'd only really use `abc > efg > hij` when telling a user where to click. And even then we'd use the right arrow symbol →.

Please use relref links: https://gohugo.io/functions/relref/#readout.